Add stdint.h to avoid build failure with GCC 15, which complains about missing symbols
for e.g. int32_t and recommends including this header.

Author: Jan Andre Reuter (jan@zyten.de)

diff --color -Naur ELFIO-Release_3.12.orig/elfio/elf_types.hpp ELFIO-Release_3.12/elfio/elf_types.hpp
--- ELFIO-Release_3.12.orig/elfio/elf_types.hpp	2023-06-02 16:37:37.000000000 +0200
+++ ELFIO-Release_3.12/elfio/elf_types.hpp	2025-06-19 21:39:21.114798603 +0200
@@ -23,6 +23,8 @@
 #ifndef ELFTYPES_H
 #define ELFTYPES_H
 
+#include <stdint.h>
+
 #ifdef __cplusplus
 namespace ELFIO {
 #endif
